src/Microsoft.AspNetCore.Authentication.Abstractions/AuthenticationOptions.cs

```diff
@@ -61,28 +61,47 @@ public void AddScheme(string name, Action<AuthenticationSchemeBuilder> configure
-        /// Used by as the default scheme by <see cref="IAuthenticationService.AuthenticateAsync(HttpContext, string)"/>.
+        /// The scheme that should be authenticated automatically to set <see cref="HttpContext.User"/>"/>.
+        /// </summary>
+        public string AutomaticAuthenticateScheme { get; set; }
+
+        /// <summary>
+        /// Used as the default scheme by <see cref="IAuthenticationService.AuthenticateAsync(HttpContext, string)"/>.
         /// </summary>
         public string DefaultAuthenticateScheme { get; set; }
 
         /// <summary>
-        /// Used by as the default scheme by <see cref="IAuthenticationService.SignInAsync(HttpContext, string, System.Security.Claims.ClaimsPrincipal, AuthenticationProperties)"/>.
+        /// Used as the default scheme by <see cref="IAuthenticationService.SignInAsync(HttpContext, string, System.Security.Claims.ClaimsPrincipal, AuthenticationProperties)"/>.
         /// </summary>
         public string DefaultSignInScheme { get; set; }
 
         /// <summary>
-        /// Used by as the default scheme by <see cref="IAuthenticationService.SignOutAsync(HttpContext, string, AuthenticationProperties)"/>.
+        /// Used as the default scheme by <see cref="IAuthenticationService.SignOutAsync(HttpContext, string, AuthenticationProperties)"/>.
         /// </summary>
         public string DefaultSignOutScheme { get; set; }
 
         /// <summary>
-        /// Used by as the default scheme by <see cref="IAuthenticationService.ChallengeAsync(HttpContext, string, AuthenticationProperties)"/>.
+        /// Used as the default scheme by <see cref="IAuthenticationService.ChallengeAsync(HttpContext, string, AuthenticationProperties)"/>.
         /// </summary>
         public string DefaultChallengeScheme { get; set; }
 
         /// <summary>
-        /// Used by as the default scheme by <see cref="IAuthenticationService.ForbidAsync(HttpContext, string, AuthenticationProperties)"/>.
+        /// Used as the default scheme by <see cref="IAuthenticationService.ForbidAsync(HttpContext, string, AuthenticationProperties)"/>.
         /// </summary>
         public string DefaultForbidScheme { get; set; }
+
+        /// <summary>
+        /// If set, the default <see cref="IAuthenticationSchemeProvider"/> will attempt to fallback logic when 
+        /// Default[Authenticate/SignIn/SignOut/Challenge/Forbid]Schemes are not set.
+        /// This is enabled by default.
+        /// </summary>
+        public bool EnableDefaultFallback { get; set; } = true;
+
+        /// <summary>
+        /// If <see cref="EnableDefaultFallback"/> and this are both true, the default <see cref="IAuthenticationSchemeProvider"/> will use a single handler as the default for 
+        /// the Default[Authenticate/SignIn/SignOut/Challenge/Forbid]Schemes.
+        /// This is enabled by default.
+        /// </summary>
+        public bool EnableDefaultSingleHandlerFallback { get; set; } = true;
```

Comment:
I think the min bar to unblock @PinpointTownes would be a way for him to opt out the single handler defaulting behavior, he only has a single ASOS handler but that throws on Authenticate, which UseAuthentication() will always call.

See https://github.com/aspnet/Security/issues/1287